### PR TITLE
add crud functions

### PR DESCRIPTION
**Pull request by disc3110**
I added a function to add thing writing and pressing the return key, you can edit or remove a selected task or all finished tasks.

### CODE REVIEW

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hello @disc3110** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

 - You did a great job, but you need a little push.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- The target for the enter button to add a task (e.keyCode === 13) is not working 
- The drag-and-drop option also it's not working
- The delete option also

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/arr-crud.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    this.tasksArr = this.tasksArr.filter((taskobj, index) => index !== idx);
+    for (let x = 0; x < this.tasksArr.length; x += 1) {
+      this.tasksArr[x].index = x + 1;
+    }
+    return this.tasksArr;
+  }
+
+  clearAll() {
+    this.tasksArr = [];
+  }
```
Comment (dasileker) [inline src/arr-crud.js]: - [ ]  The function for todolist to add and remove, it needs to target the localStorage

**Comment by disc3110** [comment]
Hi thanks for your comments.

As you can see in the video every function is working for me. you need to open it in a server for them to work as the readme says.

I added the Local Storage in the add and remove functions .

**Review by bellom — APPROVED** [review]
## Hi @disc3110  👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**